### DISCUSSION TITLE
Starting redis on Random Port

### DISCUSSION THREAD

**Discussion by amanhasank** [alicebob/miniredis#369] [Q&A]
Hi, as of now miniredis is starting redis server on a random port is there any way to predefine the port that should be used by redis server?
or can we add that functionality?
If we add below code in miniredis.go this can be achieved, Please share your views
```
func (m *Miniredis) SetPort(port int) {
	m.port = port
}
```

**Comment by alicebob** [comment]
Thanks for opening this. See #370, which answers your question.
